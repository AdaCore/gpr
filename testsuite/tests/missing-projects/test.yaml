description:
    tests error message when some project files are missing
driver: build_and_run
project_file: missing_projects
main: main
