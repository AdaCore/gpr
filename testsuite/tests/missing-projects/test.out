Messages found for demo.gpr
demo.gpr:2:6: imported project file imp_missing.gpr not found

Messages found for demo2.gpr
demo2.gpr:1:6: imports /missing-projects/demo.gpr
demo.gpr:2:6: imported project file imp_missing.gpr not found

Messages found for demo3.gpr
demo3.gpr:2:15: extends /missing-projects/demo.gpr
demo.gpr:2:6: imported project file imp_missing.gpr not found

Messages found for demo4.gpr
demo4.gpr:2:15: extended project file ext_undef.gpr not found

Messages found for demo5.gpr
demo5.gpr:2:6: imports /missing-projects/int.gpr
int.gpr:2:6: imports /missing-projects/demo.gpr
demo.gpr:2:6: imported project file imp_missing.gpr not found

